Bsii.Dotnet.Utils/DiagnosticsExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Diagnostics;
+
+namespace Bsii.Dotnet.Utils
+{
+    public static class DiagnosticsExtensions
+    {
+        public static IDisposable useActivity(this Activity activity)
```

Comment:
```suggestion
        public static IDisposable UseActivity(this Activity activity)
```
Or maybe simply `Use`?

Reply:
Agreed